index.html

```diff
@@ -652,6 +652,16 @@ <h2>Security</h2>
         <p><span class="rfc2119-assertion" id="common-constraints-security-4">
             A Thing MUST support at least one of the above security schemes.</span>
         </p>
+        <p><span class="rfc2119-assertion" id="common-constraints-security-basic-in">
+            For the <code>BasicSecurityScheme</code> the "in" field MUST be either omitted
+            or be given its default value of "header" as defined in [[wot-thing-description11].</span>
```

Comment:
I appreciate that this simplifies implementation for Consumers. However, as I mentioned in #6, an unfortunate side effect of this is that it will make it impossible to use the `BasicSecurityScheme` with the Server-Sent Events and WebSocket APIs in web browsers, since they provide no mechanism to set an `Authorization` header via these APIs.

That also effectively excludes the `BasicSecurityScheme` from being used by the HTTP SSE Profile in web application implementations.

Is this an intentional constraint?

I don't personally have a use for `BasicSecurityScheme` in these circumstances because WebThings uses JWT Bearer tokens (which for WebSockets and SSE have to be provided in a query string by the web front end), but I can imagine that some Thing implementations might prefer the simpler `BasicSecurityScheme` if the only alternative is a full OAuth2 implementation.

Reply:
I'm not sure we WANT to use basic for SSE.  To be clear, these defaults correspond to what HTTP *normally* does for BasicAuth, so no changes to implementations should be necessary.  Changes to the TD are only needed to describe what the protocol is doing.  To be honest, I will see if we can fix the issue with "name" in the TD spec prior to PR, although the variant default values for proxy/non-proxy makes it annoying.

The *right* thing to do for webhooks is bearer tokens, IMO.  For SSE I'll have to think about it, but afaik you can bootstrap off the authorization needed on the subscription affordance.